### PR TITLE
Added random number generator

### LINKED ISSUE #140: Add random number generator route to the api folder
I will add the random number generator route to the api folder

Please assign this issue to me and add the 'hacktoberfest' label to it.

### PR DESCRIPTION
**Pull request by Sukriti-m**
# Pull Request

## Description
Added random number generator

## Related Issue

Closes #140 

## Version

Did you remember to update the version number in the package.json file?

- [ ] For a major restructuring, or large changes please bump the the major version number. (ex: 1.x.x)
- [x] For a minor change, bump the minor version number. (ex: x.1.x)
- [ ] For a bug fix, bump the patch version number. (ex: x.x.1)

I am open to suggestions and changes.
@j-dogcoder  Please merge my PR and add the 'hacktoberfest-accepted' label to the PR.

### CODE REVIEW

**Review by j-dogcoder — CHANGES_REQUESTED** [review]
We should not have a package-lock.json, please delete it

**Comment by Sukriti-m** [comment]
> We should not have a package-lock.json, please delete it

I have made the required changes. Please review it,

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

**Comment by j-dogcoder** [comment]
Also, please update your branch and resolve the merge conflicts.

**Comment by j-dogcoder** [comment]
We do also need the package.json version number to be bumped. It should be `7.2.0` after this PR

File: src/routes/api/randomnumber.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import express from "express";
+import { v4 as uuidv4 } from "uuid";
```
[the remark this answers is not in the document]
Comment (Sukriti-m) [reply src/routes/api/randomnumber.ts]: I have resolved it.Please check once

**Review by j-dogcoder — APPROVED** [review]

**Comment by j-dogcoder** [comment]
LGTM

**Comment by j-dogcoder** [comment]
### 🎉 Congratulations @Sukriti-m! 🎉 

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labeled `help wanted` for additional opportunities to contribute.

**Discord Server:**
_We would love it if you joined our community discord server! Your contribution to the project make you eligible for special roles & perks._

**Pull request merged**